src/components/ProfilePage/ProfilePage.tsx

```diff
@@ -151,6 +153,9 @@ const ProfilePage = () => {
         eventList.forEach((eventId: string) => {
           const trimmedEventId = eventId.trim();
           const eventsRef = ref(database, "events");
+          if (trimmedEventId[0] == '{') {
```

Comment:
Explain this part of code. What is the reason behind adding this ?

Reply:
For some reason, eventList was having a item '{card.id}' in it which was giving error with child function. So i ignored that case

Reply:
Remove ,this check is unnecessary.